<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UtilTest</class>
 <widget class="QWidget" name="UtilTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>271</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Input curve:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Curve resolution:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="qMRMLSliderWidget" name="curveResolutionSliderWidget">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
     <property name="quantity">
      <string>length</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Slice size:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Rotation angle: </string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="qMRMLSliderWidget" name="rotationAngleSliderWidget">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
     <property name="suffix">
      <string> deg</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Output straightened volume:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="qMRMLNodeComboBox" name="outputStraightenedVolumeSelector">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="toolTip">
      <string>Pick the output to the algorithm.</string>
     </property>
     <property name="nodeTypes">
      <stringlist>
       <string>vtkMRMLScalarVolumeNode</string>
      </stringlist>
     </property>
     <property name="showChildNodeTypes">
      <bool>false</bool>
     </property>
     <property name="baseName">
      <string>Straightened volume</string>
     </property>
     <property name="noneEnabled">
      <bool>true</bool>
     </property>
     <property name="addEnabled">
      <bool>true</bool>
     </property>
     <property name="removeEnabled">
      <bool>true</bool>
     </property>
     <property name="renameEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Output projected volume:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="qMRMLNodeComboBox" name="outputProjectedVolumeSelector">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="toolTip">
      <string>Pick the output to the algorithm.</string>
     </property>
     <property name="nodeTypes">
      <stringlist>
       <string>vtkMRMLScalarVolumeNode</string>
      </stringlist>
     </property>
     <property name="showChildNodeTypes">
      <bool>false</bool>
     </property>
     <property name="baseName">
      <string>Projected volume</string>
     </property>
     <property name="noneEnabled">
      <bool>true</bool>
     </property>
     <property name="addEnabled">
      <bool>true</bool>
     </property>
     <property name="removeEnabled">
      <bool>true</bool>
     </property>
     <property name="renameEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QPushButton" name="applyButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="qMRMLNodeComboBox" name="inputCurveSelector">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip">
        <string>Pick the input to the algorithm.</string>
       </property>
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLMarkupsCurveNode</string>
        </stringlist>
       </property>
       <property name="showChildNodeTypes">
        <bool>false</bool>
       </property>
       <property name="addEnabled">
        <bool>true</bool>
       </property>
       <property name="removeEnabled">
        <bool>true</bool>
       </property>
       <property name="renameEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="qSlicerMarkupsPlaceWidget" name="markupsPlaceWidget"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Input volume:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="qMRMLNodeComboBox" name="inputVolumeSelector">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="toolTip">
      <string>Background layer of this slice view will be reformatted</string>
     </property>
     <property name="nodeTypes">
      <stringlist>
       <string>vtkMRMLScalarVolumeNode</string>
      </stringlist>
     </property>
     <property name="showChildNodeTypes">
      <bool>false</bool>
     </property>
     <property name="addEnabled">
      <bool>false</bool>
     </property>
     <property name="removeEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Slice resolution:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="qMRMLSliderWidget" name="sliceResolutionSliderWidget">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
     <property name="quantity">
      <string>length</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="qMRMLCoordinatesWidget" name="sliceSizeCoordinatesWidget">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="dimension">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="coordinates" stdset="0">
      <string>30,30</string>
     </property>
     <property name="quantity">
      <string>length</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCoordinatesWidget</class>
   <extends>QWidget</extends>
   <header>ctkCoordinatesWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLCoordinatesWidget</class>
   <extends>ctkCoordinatesWidget</extends>
   <header>qMRMLCoordinatesWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLSliderWidget</class>
   <extends>ctkSliderWidget</extends>
   <header>qMRMLSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerMarkupsPlaceWidget</class>
   <extends>qSlicerWidget</extends>
   <header>qSlicerMarkupsPlaceWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>inputVolumeSelector</tabstop>
  <tabstop>inputCurveSelector</tabstop>
  <tabstop>curveResolutionSliderWidget</tabstop>
  <tabstop>sliceResolutionSliderWidget</tabstop>
  <tabstop>sliceSizeCoordinatesWidget</tabstop>
  <tabstop>rotationAngleSliderWidget</tabstop>
  <tabstop>outputStraightenedVolumeSelector</tabstop>
  <tabstop>outputProjectedVolumeSelector</tabstop>
  <tabstop>applyButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>inputCurveSelector</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>markupsPlaceWidget</receiver>
   <slot>setCurrentNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
